validate_sptx/util.py

```diff
@@ -98,7 +105,7 @@ def validate_file(self, target_file: str) -> bool:
         target_object = self.load_json(target_file)
         return self.validate_object(target_object, target_file)
 
-    def validate_object(self, target_object: Dict, target_file: str=None) -> bool:
+    def validate_object(self, target_object: Dict, target_file: str=None, fuzz: bool=False) -> bool:
```

Comment:
I would prefer that `fuzz` be a separate method, as its purpose is quite different than `validate_object`. 

Not only that, but when fuzz is true, it doesn't actually validate any more!

Reply:
Good point!